client/pages/register.jsx

```diff
@@ -78,27 +64,41 @@ export default function RegisterPage(props) {
       Router.push("/");
     }
   };
-
-  const pickDropDown = idx => {
-    setQuestionIdx(idx);
+  const handleQuestion = questionIdx => {
+    setQuestionIdx(questionIdx);
   };
-
-  const toggle = () => {
-    setDropdownOpen(prevState => !prevState);
+  const handleCountry = country => {
+    setCountry(country);
+  };
+  const handlePerson = person => {
+    setPerson(person);
```

Comment:
since these functions simply wrap a state setter with no modifications, you can use the state setters themselves as the handler functions in your JSX